packages/svelte/src/internal/client/dom/elements/transitions.js

```diff
@@ -331,7 +335,7 @@ function animate(element, options, counterpart, t2, on_finish, on_abort) {
 		// ...but we want to do so without using `async`/`await` everywhere, so
 		// we return a facade that allows everything to remain synchronous
 		return {
-			abort: () => a.abort(),
+			abort: () => a?.abort(),
```

Comment:
this makes me a bit nervous — if this is now being called before `a` exists, then will `a` still get created (but never aborted) in the microtask?

Reply:
Good point, added a guard for it.